packages/toolbox/src/swarm.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import os from 'os'
+import keythereum from 'keythereum'
+import fs from 'fs-extra'
+
+export type KeyObject = {
+  address: string,
+  Crypto: {
+    cipher: string,
+    ciphertext: string,
+    cipherparams: {
+      iv: string,
+    },
+    mac: string,
+    kdf: string,
+    kdfparams: {
+      c: number,
+      dklen: number,
+      prf: string,
+      salt: string,
+    },
+  },
+  id: string,
+  version: number,
+}
+
+const homedir = os.homedir()
+const password = 'mainframe'
+const datadir = `${homedir}/.ethereum`
```

Comment:
Might be better to have under our own namespace. eg `.mainframe-os/services/swarm` or something?

Is this format portable to windows? `/` vs `\`

Reply:
According to the docs there
> On Windows, both the forward slash (/) and backward slash (\\) are accepted as path segment separators; however, the path methods only add backward slashes (\\).

Reply:
Better use `path.sep` then: https://nodejs.org/dist/latest-v10.x/docs/api/path.html#path_path_sep